partner_communication/models/communication_config.py

```diff
@@ -179,6 +181,19 @@ def _validate_attachment_function(self):
     ##########################################################################
     #                             PUBLIC METHODS                             #
     ##########################################################################
+    @api.model
+    def get_config_for_lang(self, lang):
+        specific_config_found = False
+        for config in self.omr_config_ids:
+            if config.lang_id:
+                if config.lang_id == lang:
+                    specific_config_found = True
+                    return config
+
+        if not specific_config_found:
```

Comment:
This test will always be true since you return from the function if a specific config is found. You can therefore remove the variable and the test and just return here the filtered function. If not config is found from the filtered function it will return an empty recordset which is fine.
```suggestion
        return self.omr_config_ids.filtered(lambda c: not c.lang_id)
```